docs/guides/03-validation-proxy.md

```diff
@@ -0,0 +1,45 @@
+# Validation Proxy
+
+Unlike mocking, the validation proxy expects there to be a real API running somewhere. This could be a localhost environment, docker container, hosted sandbox environment, or deployed to actual production, doesn't matter.
```

Comment:
Is this correct?

> the validation proxy expects there to be a real API running somewhere

Reply:
Yes that is what the upstream server is?

Reply:
>  there to be 

I didn't expect such locution to be correct, I've learned a new thing :)